Dockerfile

```diff
@@ -61,6 +61,10 @@ COPY tests /app/tests
 ENV PYTHONPATH=.
 COPY bin/startup.sh /app/startup.sh
 COPY redis.conf /app/redis.conf
+
+# Set the SWEEP_VERSION environment variable to the current date and time during image build
+ARG SWEEP_VERSION
+ENV SWEEP_VERSION=${SWEEP_VERSION:-"$(date +%Y%m%d%H%M)"}
```

Comment:
you need to run this before saving it to an envvar otherwise it's treated as a literal